client/assets/javascripts/modules_new/TextAreaAutoSize.js

```diff
@@ -0,0 +1,22 @@
+const TextAreaAutoSize = {
+  init: function (formArea) {
+    const textAreas = formArea.querySelectorAll('[class*="js-auto-size"] textarea')
+
+    textAreas.forEach(textArea => {
```

Comment:
Better to do event delegation rather than add a listener to every textarea.

Reply:
True I didn't even think about event delegation while rewriting these. Suppose that's what happens when you don't write it often anymore 😅